src/interface/variable.hpp

```diff
@@ -220,12 +220,13 @@ class FaceVariable {
   inline bool IsSparse() const { return false; }
   inline int GetSparseID() const { return InvalidSparseID; }
 
-  FaceArray<T> data;
-
  private:
-  std::array<int, 6> dims_;
   Metadata m_;
   std::string label_;
+  std::array<int, 6> dims_;
+
+ public:
+  FaceArray<T> data;
```

Comment:
Why this change?

Reply:
Sorry I had an issue with the ordering of the data in `ParticleVariable` for the initializer list and thought this would appear in `FaceVariable`, but looking again at the code this was correct in the first place. I reverted this change.